config/default.yml

```diff
@@ -794,6 +794,12 @@ RSpec/RedundantAround:
   VersionAdded: '2.19'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/RedundantAround
 
+RSpec/RedundantContext:
+  Description: Detect redundant `context` hook.
+  Enabled: pending
```

Comment:
I’m usually strongly against disabling cops, but this one, even though certainly serving some projects that strive to enforce this style, isn’t a good fit for the majority.

I support what @koic and @bquorning said.